enclave/core/sgx/keys.c

```diff
@@ -218,6 +220,38 @@ static oe_result_t _get_default_key_request_attributes(
     return result;
 }
 
+static oe_result_t _get_seal_key_info_internal(
+    oe_seal_policy_t seal_policy,
+    uint64_t tee_specifc,
```

Comment:
We use it as flags on SGX doesn't mean it has to be flags on other TEEs. I don't use `_flags` suffix here because I'd like to keep its actual meaning TEE specific.

Reply:
then change to
```
uint8_t* tee_specific buffer,
size_t tee_specific_buffer_size,
```